distribution/docs/src/main/resources/_contents/_catalog-contents/extending-catalog-contents.adoc

```diff
@@ -3670,7 +3671,7 @@ The GeoJSON input transformer is responsible for translating specific GeoJSON in
 
 ====== Installing and Uninstalling
 
-Install the catalog-rest-endpoint feature using the http://localhost:8181/admin[Admin Console].
+Install the `catalog-rest-endpoint` feature using the ${secure_url}/admin[Admin Console].
```

Comment:
Square brackets after a url set the link text, which I think is maybe wrong here, because elsewhere we don't link to the `localhost` url in case it has been changed from the default.